resources/update-route53/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		log.Fatalln(errors.New("Wrong number of arguments. Arguments must be non-empty strings passed into 'route53-updater <service-role> <internal-hosted-zone-name>'"))
+	}
+	serviceRole := os.Args[1]
+	internalHostedZoneName := os.Args[2]
+	if serviceRole == "" || internalHostedZoneName == "" {
+		log.Fatalln(errors.New("Service role or internal hosted zone name missing. Both arguments must be non-empty strings passed into 'route53-updater <service-role> <internal-hosted-zone-name>'"))
+	}
+	internalIP, err := exec.Command("curl", "-fsSL", "http://169.254.169.254/latest/meta-data/local-ipv4").Output()
```

Comment:
does it make sense to run this as an external command or just make an http request here?